### PR TITLE
Add a README section for the remote terminal functionality

### PR DESCRIPTION
**Pull request by oleorhagen**
Also remote the `Pre-alpha` metadata from setup.py

### CODE REVIEW

**Comment by oleorhagen** [comment]
As we discussed @lluiscampos. The words were not really pouring out of me this morning though, you think it's too short?

File: README.md [hunk 1]
```diff
@@ -25,6 +29,24 @@ sub-updater on the device has handled the update, reported the update status
  architecture is impossible.
 
 
+#### Remote terminal
+
+It also has support for remote terminal connectivity, as described
+[here](https://docs.mender.io/add-ons/remote-terminal).
+
+In short, this allows you to connect to the client via the Mender UI, and
+connect to your client as if you had an ssh server running on the device.
```
Comment (lluiscampos) [inline README.md:38]:
```suggestion
In short, this allows you to connect to the client via the Mender UI or `mender-cli`, and
connect to your device as if you had an ssh server running on the device.
```

**Review by lluiscampos — APPROVED** [review]
I think the length is fine, we don't need much more. I first thought we could point to the configuration parameters from [here](https://docs.mender.io/add-ons/reference#configuration), but then I realized that we only implement a subset of it, so better to keep it a bit vague...

Minor comment below.

**Comment by oleorhagen** [comment]
Skipping the integration test - not needed.

**Pull request merged**